KebabCaseConvrter/Program.cs

```diff
@@ -1,12 +1,29 @@
 ﻿using System;
-
+using System.Text;
 namespace KebabCaseConvrter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Enter your text: ");
+            string regularText = Console.ReadLine();
+
+            string[] words = regularText.Split(new char[] { ',', ' ', '.', ';', '-' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Try to use var instead of strong typing where it's possible. It's optional in your case, but you can read about another cases here https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/var .

Also move delimiters array for Split method to a variable.